CHANGELOG.md

```diff
@@ -7,5 +7,10 @@
   - Add QBS template
   - Add all environment variables to README
 
+### console 2.0
```

Comment:
Just leave these in the "unreleased" block for now - they'll get tagged as 2.0.0 when the release is actually taken.